graylog2-web-interface/src/components/indices/IndexSetFieldTypeProfiles/cells/IndexSetsCell.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+import { styled } from 'styled-components';
+
+import { Link } from 'components/common/router';
+import Routes from 'routing/Routes';
+import useCurrentUser from 'hooks/useCurrentUser';
+import { isPermitted } from 'util/PermissionsMixin';
+
+const List = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 5px;
+`;
+
+const IndexSetsCell = ({ indexSetIds, normalizedIndexSetsTitles }: { indexSetIds : Array<string>, normalizedIndexSetsTitles: Record<string, string> }) => {
+  const currentUser = useCurrentUser();
+
+  if (!isPermitted(currentUser.permissions, ['indexsets:read'])) {
+    return null;
+  }
+
+  return (
+    <List>{indexSetIds
+      .map((indexSetId) => (
+        <Link key={indexSetId}
+              to={Routes.SYSTEM.INDEX_SETS.SHOW(indexSetId)}
+              target="_blank">
+          {normalizedIndexSetsTitles[indexSetId]}
+        </Link>
+      ),
+      )}
```

Comment:
In similar cases we display the items separated by a comma, instead of just a gap.

<img width="290" alt="image" src="https://github.com/Graylog2/graylog2-server/assets/46300478/c558eb1e-0d1d-4164-bf13-7f8df9d4b33e">